### DISCUSSION TITLE
[Feature]: Don't select "Recent" by default

### DISCUSSION THREAD

**Discussion by jj-style** [immich-app/immich#1592] [Feature Request]
### Feature detail

On the mobile app one or two times between app upgrades, or when I've cleared storage of the app and re-logged in it auto-selects "Recent" from albums and starts automatically backing up.

This is quite annoying as I like to just back up DCIM from my phone and exclude folders like WhatsApp images but by the time I get to settings and select DCIM and disable Recent too many pictures have uploaded and cluttered my library and I end up having to reset immich and start from scratch.

It would be great if the app either
- didn't select any album by default / asked on first open which album should be selected
- doesn't enable foreground backup to allow the user to go to settings and switch albums before the first backup kicks off

### Platform

Mobile App

**Comment by bo0tzz** [comment]
I think this might be related to #1526, where your "foreground backup enabled" setting is being stored on the server. Until that issue is closed, a workaround should be to just disable backup before resetting things.

**Comment by jj-style** [comment]
Ah thank you I hadn't spotted that open issue. Yeah that is definitely a workaround for now. Thank you!
